rs/moq-native/src/server.rs

```diff
@@ -78,10 +79,36 @@ impl ServerConfig {
 	}
 }
 
+#[derive(Clone)]
+pub struct ServerReloader {
+	certs: Arc<ServeCerts>,
+}
+
+impl ServerReloader {
+	pub fn reload(&self, config: &ServerTlsConfig) -> anyhow::Result<()> {
```

Comment:
Another option is to add this method to `ServeCerts` instead, so it will load and replace without this `ServerReloader`. In that scenario, then you would need a RWMutex.